### ISSUE TITLE
insertStatistics issues

### ISSUE THREAD

**Issue by gasparl** [WillemSleegers/tidystats-Google-Docs-add-in#5]
In the current implementation of insertStatistics, I think some things are missing:
- More statistics with df: `["t", "χ²", "K²", "CMH", "OR", "Q", "QM", "QE", "QS", "LRT"].includes(statistic.symbol)` ...
https://github.com/gasparl/tidystats-Google-Docs-add-in/blob/master/src/server/functions/insertStatistics.ts#L93
- The special case of ηG², where G should be lowered: `if (newText == "ηG²") {` ...
https://github.com/gasparl/tidystats-Google-Docs-add-in/blob/master/src/server/functions/insertStatistics.ts#L31
- Generally, your solution checks for specific instances of `statistic.name === "statistic"`, OR anything else. This seems both superfluous and problematic because even if `statistic.name === "statistic"` is true, additional criteria needs to be met (e.g., statistic.symbol === "t") in which cases `statistic.name === "statistic"` is always true anyway, while in some cases statistic.name === "statistic" may be true, but the symbol is not included, hence nothing is inserted. I think all this could simply be replaced with the way I wrote it (starting at https://github.com/gasparl/tidystats-Google-Docs-add-in/blob/master/src/server/functions/insertStatistics.ts#L92).

**Comment by WillemSleegers** [comment]
Yes, I copied what I had initially written so it's not updated with the changes you made while working on the google doc.
